apps/mobile/src/components/TradeChecklistFlow/components/AgreeOnTradeDetailsScreen/components/ChecklistCell.tsx

```diff
@@ -104,11 +112,6 @@ function ChecklistCell({item}: Props): JSX.Element {
           </Stack>
         </XStack>
         <XStack ai={'center'} space={'$2'}>
-          {/* {sideNote && (
```

Comment:
Check if side note should be displayed based on Figma design. If so create a ticket for this